tests/test-vic-serial.c

```diff
@@ -19,12 +19,47 @@ static char * test_process()
 	return 0;
 }
 
+static char * test_process_backspace()
+{
+	char in[] = "call\n";
 
-static char * all_tests()
+	int i;
+	for(i = 0; i < strlen(in); i++){
+		vic_process(in[i]);
+	}
+	for(i = 0; i < strlen(in) + 2; i++){
+		vic_process((char)0x08);
```

Comment:
@vargac This line would be way more obvious if there was a comment on what `0x08` stands for.